src/core/tsi/ssl_transport_security.h

```diff
@@ -332,4 +332,8 @@ const tsi_ssl_handshaker_factory_vtable* tsi_ssl_handshaker_factory_swap_vtable(
     tsi_ssl_handshaker_factory* factory,
     tsi_ssl_handshaker_factory_vtable* new_vtable);
 
+/* Exposed for testing only. */
+tsi_result extract_x509_subject_names_from_pem_cert(const char* pem_cert,
```

Comment:
Since it is header, we need proper namespace.
s/extract_x509_subject_names_from_pem_cert/tsi_ssl_extract_x509_subject_names_from_pem_cert

Reply:
Thanks Jiangtao. All comments are addressed. PTAL.